File: app/src/main/java/com/sermage/mymoviecollection/pojo/Reviews.java
package com.sermage.mymoviecollection.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Reviews {
    @SerializedName("author")
    @Expose
    private String author;
    @SerializedName("content")
    @Expose
    private String content;


    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }


}

File: app/src/main/java/com/sermage/mymoviecollection/pojo/Reviews.kt
package com.sermage.mymoviecollection.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Reviews (
    @SerializedName("author")
    @Expose
    val author: String? = null,

    @SerializedName("content")
    @Expose
    val content: String? = null
)